kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -63,7 +63,15 @@
                 {{ username }}
               </p>
               <p :style="{ color: $themeTokens.annotation, fontSize: '12px', marginTop: 0 }">
-                {{ getUserKind }}
+                <span v-if="getUserKind === 'learner'">
```

Comment:
Could you move this conditionality into a computed prop on the component?

```
userKindText() {
if (this.getUserKind === UserKind.LEARNER) {
    return this.coreString('learnerLabel');
}
....
```
etc.

Then can just reference `userKindText` in the template. The main reason for this is to avoid hard coding the values into the template (and I _think_ `'super admin'` might not be the correct text for the user kind).

I think we probably also need to add a condition for 'admin' too.